variables.js

```diff
@@ -0,0 +1,20 @@
+let adminName ='Maria';
+let userName ='Igor';
+
+console.log(typeof adminName, typeof userName);
+
+let feedback = "awesome";
```

Comment:
Pls use the same quotes style for your all code. 

```suggestion
let feedback = 'awesome';
```